MultiSource/Benchmarks/Ptrdist/ft/ft.c

```diff
@@ -89,30 +91,30 @@ main(int argc, _Array_ptr<const char*> argv : count(argc) )
   }
 
   if(debug)
-  {
+  _Unchecked {
```

Comment:
You might want to introduce a checked scope after the argument manipulation is done.